### DISCUSSION TITLE
Daily tasks.

### DISCUSSION THREAD

**Discussion by Adaeze-69** [chiche-ds/30-Day-Terraform-challenge-#22] [Q&A] [answered]
Are we to fork the repository everyday for the daily tasks?

**Comment by Adaeze-69** [comment]
Also,the videos you asked us to watch on Udemy (video 7-10),the topics attached to them doesn't match what is on Udemy.

**Reply by chiche-ds** [reply, marked as answer]
thanks to your comment we doing our best that they do match now. Here is your workflow everyday 

1. sync fork
2. create branch 
3. go to dayX folder 
4. Read the README.md 
5. Do all task 
6. create  dayx-update-yourusername.md 
7. commit file to dayx branch 
8. submit your pull request
